utils/python_lint.py

```diff
@@ -23,9 +23,17 @@ def lint(arguments, verbose=True):
     )
     if flake8_result != 0:
         if verbose:
-            print("Missing modules flake8 or flake8-import-order. Please be"
-                  " sure to install these python packages before linting.")
-        return 0
+            print("""
+The flake8 and flake8-import-order Python packages are required for linting,
+but these were not found on your system.
+
+You can install these using:
+
+    python -m pip install flake8
+    python -m pip install flake8-import-order
+
+For more help, see http://flake8.pycqa.org.""")
+        return flake8_result
```

Comment:
You can keep the message, but it seems that our Linux bots lint gyb files and don’t have flake8 installed.  Please keep the return value as is.

Reply:
Thank you for catching this with a smoke test.

I've reverted the change, and added a sticky note.

With this reverted, there are no functional changes. So the commit message for the squash could read: "Improve Python linter documentation".